content/sensu-core/1.7/reference/clients.md

```diff
@@ -112,8 +112,10 @@ registration events are logged in the [Sensu server][5] log._
 
 Similarly to registration events, Sensu can create and process a deregistration event when the Sensu client process stops.
 You can use deregistration events to trigger a handler that updates external CMDBs or performs an action to update ephemeral infrastructures.
-By default, registration events trigger the built-in deregistration extension.
-To enable deregistration events, set the client `deregister` flag to `true` and specify the event handler using the client [`deregistration` definition scope][48].
+
+By default, registration events trigger the built-in deregistration extension. To enable deregistration events, set the client `deregister` flag to `true`. This will result in clients being immediately depopulated from the dashboard. If you require an external servide to be handle the deregistration event (and specify the event handler using the client [`deregistration` definition scope][48].
```

Comment:
```suggestion
By default, registration events trigger the built-in deregistration extension. To enable deregistration events, set the client `deregister` flag to `true`. This will result in clients being immediately depopulated from the dashboard. If you require an external service to be handle the deregistration event (and specify the event handler using the client [`deregistration` definition scope][48].
```

Reply:
Looks like there's also an errant `(` here.